data/discord_messages.yaml

```diff
@@ -1,3 +1,1664 @@
+    `configuration.yaml`
+
+    2. Restart Home Assistant.
+
+
+    _Sidenote: this will cause you to lose all your history, but it's broken
+    anyway, so just do it._
+works:
+  content: >-
+    Want to know if something works with Home Assistant? Here's a few things to
+    try:
+
+
+    1. Look at the [integrations
+    list](https://www.home-assistant.io/integrations)
+
+    2. Search [community forum](https://community.home-assistant.io/)
+
+    3. Try Googling _Home Assistant MYDEVICE_ or _MYDEVICE Python_
+pinned:
+  content: See the pinned messages
+flooding:
+  content: >-
+    **[Rule
+    #6](https://discordapp.com/channels/330944238910963714/331130181102206976/461249903478439937):**
+    Spam will not be tolerated, including but not limited to: self-promotion,
+    ***flooding*** (posting lots of shorter messages), text walls (longer than
+    15 lines) and unapproved bots.
+
+
+    *Please take the time now to review all of the rules and references in
+    <#331130181102206976>.*
+hadebug:
+  content: >-
+    To enable debug for the deCONZ integration in Home-Assistant, add the
+    following to your configuration.yaml:```yaml
+
+    logger:
+      default: info
+      logs:
+        pydeconz: debug
+        homeassistant.components.deconz: debug
+    ```
+features:
+  content: >-
+    If you have a feature request for the frontend you can open [one
+    here](https://github.com/home-assistant/frontend/issues), for Home Assistant
+    itself please [post on the
+    forum](https://community.home-assistant.io/c/feature-requests/13). All other
+    feature requests should be made to the developer of that custom
+    card/component.
+weather:
+  content: >-
+    There is no universal _best_ [weather
+    integration](https://www.home-assistant.io/integrations#weather). Which one
+    works best _for you_ depends on where you live - try them all and see which
+    is generally the most accurate.
+best:
+  content: >-
+    There is no universal _best_ anything. There's always trade-offs, you need
+    to explain:
+
+
+    * Your priorities/desired features
+
+    * Where in the world you are
+
+    * Your budget
+
+    * Any protocols that you'd prefer (eg Zigbee) or want to avoid (eg Zigbee)
+unique_id:
+  content: >-
+    Please see the [faq
+    entry](https://www.home-assistant.io/faq/#this-entity-does-not-have-a-unique-id)
+    about that message.
+swag:
+  content: >-
+    [Secure Web Application Gateway](https://docs.linuxserver.io/general/swag)
+    aka SWAG - is the easiest way of setting up a reverse proxy server with SSL
+    (using Certbot and LetsEncrypt) for your Docker based install. See [their
+    documentation](https://docs.linuxserver.io/general/swag) on it, and if you
+    get stuck they also have [a Discord server](https://discord.gg/YWrKVTn) and
+    [forum](https://docs.linuxserver.io/general/swag).
+tts:
+  content: >-
+    If `tts` isn't working in your media players (and on a Google device at most
```

Comment:
@cogneato once this PR is merged, can you go through these snippets and update them to latest knowledge/UI. For example, this one needs to use a My link to point at https://my.home-assistant.io/redirect/network/ instead.